ios/FluentUI/SegmentedControl/SegmentedControl.swift

```diff
@@ -60,19 +70,23 @@ open class SegmentedControl: UIView, TokenizedControlInternal {
         }
     }
 
-    // Hierarchy for pill styles:
+    // Hierarchy:
     //
-    // pillContainerView (used to create 16pt inset on either side)
-    // |--backgroundView (fill container view, uses restTabColor)
-    // |--buttons (uses restLabelColor)
-    // |--pillMaskedLabelsContainerView (fill container view, uses selectedTabColor)
-    // |  |.mask -> selectionView
-    // |  |--pillMaskedLabels (uses selectedLabelColor)
-    // |  |--pillMaskedImages (uses selectedLabelColor)
+    // .mask -> gradientMask
```

Comment:
What does the . mean?

Reply:
`.mask` is the mask of the `SegmentedControl` itself, so same as `self.mask`. Currently being set on 187.